### ISSUE TITLE
glooctl dashboard doesn't work with custom namespace

### ISSUE THREAD

**Issue by bcollard** [solo-io/gloo#4716]
```bash
# Gloo Edge
export NS="not-gloo-system"
GLOO_VERSION=1.7.5
helm upgrade -i gloo glooe/gloo-ee --namespace ${NS} --version ${GLOO_VERSION} \
  --create-namespace --set-string license_key=$LICENSE_KEY

# Gloo Fed
FED_NS="not-gloo-fed"
kubectl create namespace ${FED_NS}
helm install -n ${FED_NS} gloo-fed gloo-fed/gloo-fed --set license_key=$LICENSE_KEY

glooctl ui -n not-gloo-fed
```

Output:
```
No Gloo dashboard found as part of the installation in namespace not-gloo-fed. The full dashboard is part of Gloo Enterprise by default. Error: deployments.apps "gloo-fed-console" not found
```

**Additional context**
Add any other context about the problem here, e.g.
- Gloo Edge version: v1.7.5
- Kubernetes version: GKE 1.18

**Comment by jenshu** [comment]
this may be fixed with https://github.com/solo-io/gloo/pull/5054, need to verify

**Comment by chrisgaun** [comment]
This is fixed in 1.8
